bundles/CoreBundle/Manager/ClassesManager.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace Pimcore\Bundle\CoreBundle\Manager;
+
+use Pimcore\Model\DataObject\ClassDefinition;
+
+/**
+ * Class ClassesManager
+ * @package Pimcore\Bundle\CoreBundle
+ */
+class ClassesManager
```

Comment:
I guess the class should be somewhere here: https://github.com/pimcore/pimcore/tree/master/lib/DataObject

Reply:
Thank you! Good hint, will do so.